module/zfs/dsl_scan.c

```diff
@@ -3449,8 +3446,7 @@ scan_io_queue_vacate_cb(range_tree_t *rt, void *arg)
 {
 	dsl_scan_io_queue_t *queue = arg;
 	void *cookie = NULL;
-	while (avl_destroy_nodes(&queue->q_exts_by_size, &cookie) != NULL)
-		;
+	while (avl_destroy_nodes(&queue->q_exts_by_size, &cookie) != NULL);
```

Comment:
its my preferred style, but im not looking style right now :). I will rebase everything and clean it up once I'm sure the new range tree is't causing any more problems.